content/custom_exporter/Java/Trace.md

```diff
@@ -0,0 +1,281 @@
+---
+title: "Trace exporter"
+draft: false
+weight: 3
+---
+
+### Table of contents
+- [Introduction](#introduction)
+- [Implementation](#implementation)
+- [Runnable example](#runnable-example)
+- [Notes](#notes)
+- [References](#references)
+
+#### Introduction
+
+A trace exporter must extend the abstract class [SpanExporter.Handler](https://static.javadoc.io/io.opencensus/opencensus-api/0.15.0/io/opencensus/trace/export/SpanExporter.Handler.html) implementing the `export` method
+
+which for purposes of brevity is:
+
+```java
+import java.util.Collection;
+
+import io.opencensus.trace.export.SpanData;
+
+public void export(Collection<SpanData> spanDataList);
+```
+
+The sole method `export` will be used to process and translate a collection of [io.opencensus.trace.export.SpanData](https://static.javadoc.io/io.opencensus/opencensus-api/0.15.0/io/opencensus/trace/export/SpanData.html) to your desired trace backend's data.
```

Comment:
nit: s/io.opencensus.trace.export.SpanData/SpanData. You don't need to include the full package path.

Reply:
Aye aye, I'll fix this.

Reply:
Done PTAL.